tests/Aspire.RabbitMQ.Client.Tests/Aspire.RabbitMQ.Client.Tests.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Shared\RequiresDockerTheoryAttribute.cs" />
```

Comment:
We have a shared test library. Let's move this there instead so all these projects don't need to compile include the file. You can also put it in an existing namespace like `Aspire` or `Aspire.Components`, so all the files don't need to include a new one-off namespace.

Reply:
Where is this shared test library? I don't see a csproj in `tests/Shared`.

Reply:
https://github.com/dotnet/aspire/tree/main/tests/Aspire.Components.Common.Tests